### PR TITLE
Added Longest Common Subsequence Rules

### PR DESCRIPTION
**Pull request by jrg94**
Feel free to take a look and let me know if you like these rules! I plan to start adding these for my own benefit, but I'd like a pair of eyes to help decide if these are reasonable.

### CODE REVIEW

**Comment by jrg94** [comment]
Fixed the merge conflict!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -293,6 +286,42 @@ As a result, the rules are pretty simple. For each language, create a program
...
+```
+A = [1, 4, 5, 3, 15, 6]
+B = [1, 7, 4, 5, 11, 6]
+```
+
+The longest common subsequence is `[1, 4, 5, 6]`. 
+
+Write a program which accepts two command line arguments--each list--and outputs the longest 
+common subsequence between the two lists. Input arguments should be in array notation:
+`[1, 2, 14, 11, 31, 7, 9]`.
```
Comment (jrg94) [inline CONTRIBUTING.md]: I might change this to a comma separated string based on #512, so hold tight.

Comment (chrboe) [reply CONTRIBUTING.md]: Fwiw, I think we should definitely change it. Parsing this array notation would be a nightmare in languages that don't natively support it (okay, it's not **that** bad, but a comma separated list is definitely more convenient)

Comment (jrg94) [reply CONTRIBUTING.md]: Doing that now!

**Pull request merged**